README.md

```diff
@@ -64,6 +64,16 @@ In thirdparty/opus/build: `make`
 
 In the project root: `scons`
 
+### Mac
+
+Not sure on the status of whether unsigned GDExtensions can be loaded into MacOS, but it builds, so here are the instructions anyways
+
+In thirdparty/opus: `cmake -Bbuild -DCMAKE_OSX_ARCHITECTURES="arm64;x86_64"`
```

Comment:
When I tested this on mac m1 Sonoma 14.3, I got the warning `ERRORRuntime cpu capability detection needed for MAY_HAVE_NEON`, and the `make` command failed during `Building C object CMakeFiles/opus.dir/celt/arm/celt_neon_intr.c.o`.

Might want to only build for the local machine (not universal) until that issue is resolved.

```suggestion
In thirdparty/opus: `cmake -Bbuild
```